project/SparkBuild.scala

```diff
@@ -415,6 +415,11 @@ object SparkBuild extends PomBuild {
     outputStrategy in run := Some (StdoutOutput),
 
     javaOptions += "-Xmx2g",
+    javaOptions += {
+      val versionParts = System.getProperty("java.version").split("[+.\\-]+", 3)
+      var major = versionParts(0).toInt
+      if (major >= 16) "--add-modules=jdk.incubator.vector" else ""
```

Comment:
Do you think `Java 17 (LTS)` has this an official module (not `incubator`)?

Reply:
No, it doesn't yet and it's unlikely to happen before Project Valhalla (inline/value types) is going to land into JDK mainline.